Components/cards/cards.html

```diff
@@ -0,0 +1,237 @@
+            />
+
+            <div class="">
+              <h3 class="card-heading">Nike Men's Flex 2020 Rn Running Shoe</h3>
+              <small class="card-subHeading">Sport Shoes</small>
+            </div>
+
+            <p class="card-description text-bold">₹ 10,999</p>
+          </div>
+
+          <div class="card-footer">
+            <button class="btn btn-ecommerce">
+              <span class="icon"><i class="fa fa-shopping-cart"></i></span> Add
+              to Cart
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
+        <div class="card-basic Card-horizontalImage-textContainer">
+          <div class="Card-horizontalImage-text">
+            <img
+              class="card-img Card-horizontalImage-text-img"
+              src="https://media.macphun.com/img/uploads/customer/blog/347/15361508215b8fcd25b13f46.53184133.jpg?q=85&w=1680"
+            />
+
+            <div class="">
+              <h3 class="card-heading">Our Changing Planet</h3>
+              <small class="card-subHeading">by Kurt Wagner</small>
+              <p class="card-description Card-horizontalImage-text-description">
+                Visit ten places on our planet that are undergoing the biggest
+                changes today.
+              </p>
+            </div>
+          </div>
+
+          <div class="card-footer">
+            <button class="card-btn">READ</button>
+            <button class="card-btn">BOOKMARK</button>
+            <img
+              class="heart-icon"
+              src="https://img.icons8.com/fluency-systems-regular/50/000000/like--v1.png"
+            />
+            <img src="../icons/share.png" class="" />
+            <img src="../icons/more.png" class="" />
+          </div>
+        </div>
+
+        <div class="card-basic Card-horizontalImage-textContainer">
+          <div class="Card-horizontalImage-text">
+            <img
+              class="card-img Card-horizontalImage-text-img Card-horizontalImage-text-img-Ecommerce"
+              src="https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80"
+            />
+
+            <div class="">
+              <h3 class="card-heading">Nike Men's Flex 2020 Rn Running Shoe</h3>
+              <small class="card-subHeading">Sport Shoes</small>
+              <p class="card-description text-bold">₹ 10,999</p>
+            </div>
+          </div>
+
+          <div class="card-footer">
+            <button class="btn btn-text-icon btn-ecommerce">
+              <span class="icon"><i class="fa fa-shopping-cart"></i></span> Add
+              to Cart
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
```

Comment:
it would be better if u specify which container it is like card container

Reply:
okay i will do that